server/schedule/operator/operator.go

```diff
@@ -75,6 +77,7 @@ func NewOperator(desc, brief string, regionID uint64, regionEpoch *metapb.Region
 		steps:       steps,
 		status:      NewOpStatusTracker(),
 		level:       level,
+		Counters:    make([]prometheus.Counter, 0),
```

Comment:
```suggestion
		Counters:    []prometheus.Counter,
```

Reply:
It will build failed `type []prometheus.Counter is not an expression`

Reply:
I think you can use the default value or remove this line?